plugin/evm/vm.go

```diff
@@ -245,6 +252,10 @@ type VM struct {
 	ethTxGossipHandler p2p.Handler
 	ethTxPushGossiper  avalancheUtils.Atomic[*gossip.PushGossiper[*GossipEthTx]]
 	ethTxPullGossiper  gossip.Gossiper
+
+	skipStandaloneDB bool
```

Comment:
I don't think we should have a flag like this, here's an idea https://github.com/ava-labs/subnet-evm/pull/1363 to check for memdb instead.

Reply:
applied with minor change